content/millennium/dstu2/care-provision.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Care Provision
```

Comment:
Why are we needing to add the DSTU 2 throughout?

Reply:
I added this to differentiate the DSTU 2 vs R4 page titles. of note that https://fhir.cerner.com/millennium/dstu2/devices/device/ already has " | DSTU 2 API" in its page title